### PR TITLE
docs(release): suggest running `xattr` on release tarball

### PR DESCRIPTION
**Pull request by carlocab**
This should help resolve at least some issues about unknown developers
when using the release tarball.

@clason

### CODE REVIEW

**Comment by clason** [comment]
Maybe you could add a short note in parentheses what step 2 is for?

**Comment by carlocab** [comment]
Could do -- but I find that being as terse as possible for things like this is better. That makes it more likely that people actually read and follow the directions (instead of ignoring them because they were too long to read).

**Comment by clason** [comment]
Definitely keep it terse, but asking people to enter a little-known terminal command without explanation is asking a bit much (and sets a dangerous precedent ;))

Maybe add a FAQ to the Wiki, and link to that, e.g., "(optional; see [FAQ])"?

**Comment by clason** [comment]
@justinmk ?

File: .github/workflows/notes.md [hunk 1]
```diff
@@ -21,8 +21,9 @@ ${NVIM_VERSION}
 ### macOS
 
 1. Download **nvim-macos.tar.gz**
-2. Extract: `tar xzvf nvim-macos.tar.gz`
-3. Run `./nvim-macos/bin/nvim`
+2. Run `xattr -c ./nvim-macos.tar.gz`
```
Comment (justinmk) [inline .github/workflows/notes.md:24]:
```suggestion
2. Run `xattr -c ./nvim-macos.tar.gz` (to avoid "unknown developers" warning)
```

**Pull request merged**